src/types/markers.js

```diff
@@ -413,8 +413,7 @@ export type DOMEventMarkerPayload = {|
   interval: 'start' | 'end',
   eventType: string,
   phase: 0 | 1 | 2 | 3,
-  docShellId?: string,
-  docshellHistoryId?: number,
+  innerWindowID?: number,
```

Comment:
Do we have a good place to document this? I'm guessing it'll be repeated across multiple payloads.

Reply:
It's probably good enough to just grep and find the Page object.